packages/lex-cli/src/codegen/client.ts

```diff
@@ -28,9 +27,9 @@ export async function genClientApi(schemas: Schema[]): Promise<GeneratedAPI> {
   const api: GeneratedAPI = { files: [] }
   const nsidTree = schemasToNsidTree(schemas)
   for (const schema of schemas) {
-    if ('xrpc' in schema) {
+    if (schema.type === 'query' || schema.type === 'procedure') {
```

Comment:
nice i like this better 👌

Reply:
Yeah this unification really cleaned up some things